addon/adapters/github-compare.js

```diff
@@ -0,0 +1,12 @@
+import GithubAdapter from 'ember-data-github/adapters/github';
+
+export default GithubAdapter.extend({
+  urlForQueryRecord(query) {
+    const { repo, base, head } = query;
+    delete query.repo;
+    delete query.base;
+    delete query.head;
```

Comment:
I like this approach of deleting it when the only thing that uses it has no need for it anymore.